code/modules/mob/living/simple_animal/abnormality/aleph/army_in_black.dm

```diff
@@ -0,0 +1,498 @@
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "armyinpink"
+	icon_living = "armyinpink"
+	icon_dead = "armyinpink_heart"
+	pixel_x = -16
+	base_pixel_x = -16
+
+	//*--Suppression info--*
+	maxHealth = 450
+	health = 450
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 1.0, PALE_DAMAGE = 0.8)//the same as hostile mobs, ez records
+	speed = 1//abno itself shouldn't have any stats
+	move_to_delay = 5
+	generic_canpass = FALSE
+	del_on_death = TRUE
+	is_flying_animal = TRUE
+	faction = list("hostile")
+
+	//melee stats(unused)
+	attack_sound = 'sound/abnormalities/armyinblack/pink_heal.ogg'
+	attack_verb_continuous = "shoots"
+	attack_verb_simple = "shoot"
+
+	//*--Containment info--*
+	threat_level = ALEPH_LEVEL
+	can_breach = TRUE
+	start_qliphoth = 3
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 0,
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 40, 50, 50),
+						ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 50, 55, 55),
+						ABNORMALITY_WORK_REPRESSION = 30,
+						"Protection" = 0//shouldn't attempt to generate any PE
+						)
+	work_damage_amount = 17
+	work_damage_type = WHITE_DAMAGE
+
+	//E.G.O list
+	ego_list = list(
+		/datum/ego_datum/weapon/pink,
+		/datum/ego_datum/armor/pink
+		)
+	gift_type =  /datum/ego_gifts/pink
+
+	//Unique variables
+	var/death_counter = 0
+	var/protection_duration = 120 SECONDS
+	var/protected_targets = list()
+	var/friend_army = list()//friendly unit list
+	var/summoned_army = list()//hostile unit list
+	var/boom_radius = 20
+	var/boom_damage = 70
+	var/adds_max = 1
+
+/***Simple mob procs***/
+//checks for deaths
+/mob/living/simple_animal/hostile/abnormality/army/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/OnMobDeath)
+	adds_max = clamp((LAZYLEN(GLOB.player_list)/ 2),2, 5)//between 2 and 5 mooks on breach, one for every 2 people.
+
+//stops the previous snippet from destroying the server
+/mob/living/simple_animal/hostile/abnormality/army/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/army/Life()
+	..()
+	if(LAZYLEN(protected_targets))
+		for(var/mob/living/carbon/human/H in protected_targets)
+			if(!H.has_status_effect(STATUS_EFFECT_PROTECTION))
+				protected_targets -= H
+	return
+
+//no more nuzzling
+/mob/living/simple_animal/hostile/abnormality/army/AttackingTarget()
+	return FALSE
+
+/mob/living/simple_animal/hostile/abnormality/army/Move()
+	return FALSE
+
+/***Work procs***/
+//protect work grants you a buff in exchange for reducing its counter
+/mob/living/simple_animal/hostile/abnormality/army/AttemptWork(mob/living/carbon/human/user, work_type)
+	..()
+	if(work_type == "Protection")
+		if(datum_reference?.qliphoth_meter > 1)
+			if(user in protected_targets)
+				return FALSE
+			datum_reference.qliphoth_change(-1)
+			if((get_user_level(user)) < 4)
+				user.adjustSanityLoss(500)
+				return FALSE
```

Comment:
Comically, if we're to go by LC records, *this should still allow them to get the buff*. Army in Pink merely follows them while they're insane, which is wild. Though I'll say that's definitely optional.

Reply:
The problem is that the healing will cure insanity, rather quickly I'd add. Since we don't have proper possession mechanics in place, I'd rather leave it as is.